README.md

```diff
@@ -99,12 +99,28 @@ Samila is a generative art generator written in Python, Samila let's you create
 
 ## Usage
 
+### Magic
```

Comment:
What about `Random`?
We can also don't use any header here as our main method to use Samila is this.

Reply:
I think `magic` is better, it is more fancy